functions/bouncer/src/middleware/stripeEventCustCreated.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-console */
+import debug from 'debug';
+import * as Sentry from '@sentry/node';
+import { custCreatedValidate, custMetadataValidate } from '../validate/stripe';
+
+const dlog = debug('that:api:functions:bouncer:stripeEventCustCreatedMw');
+
+export default function stripeEventCustCreated(req, res, next) {
+  dlog('stripe event customer created called');
+
+  const { whRes, stripeEvent } = req;
+  whRes.stages.push('stripeEventCustCreated');
+
+  if (stripeEvent.type !== 'customer.created') {
+    dlog('next(), not type customer.created');
+    return next();
+  }
+
+  return Promise.all([
+    custCreatedValidate(stripeEvent).catch(err => {
+      Sentry.captureException(err);
+      return {
+        name: 'custCreatedValidate',
+        status: 400,
+        err,
+      };
+    }),
+    custMetadataValidate(stripeEvent).catch(err => {
+      Sentry.captureException(err);
+      return {
+        name: 'custMetadataValidate',
+        status: 200,
+        err,
+      };
+    }),
+  ])
+    .then(result => {
+      const errors = result.reduce((acc, cur) => {
+        if (cur.name) {
+          acc.push(cur);
+        }
+        return acc;
+      }, []);
+      if (errors.length > 0) {
+        const [value] = errors;
+        console.log(`error validating ${value.name}`);
```

Comment:
dlog?

Reply:
This is gcloud function and we explicitly want this message (and others like it) to end up in the functions logs for troubleshooting assistance.